### DISCUSSION TITLE
Not seeing mirrored data

### DISCUSSION THREAD

**Discussion by jrsmsn** [Security-Onion-Solutions/securityonion#7534] [Unsupported Versions] [answered]
I installed Security Onion on a R610 server in my home lab.  I have the management port configured to [nic0], and the monitor port set to [nic3].  I have a Cisco 3750G switch that I set up a SPAN session on to mirror all ports to port #1 on the switch.  I hooked up my laptop directly to the ethernet cable running from the destination SPAN port, to the R610 port which is set to be the monitor interface and I can see all of the networks traffic in Wireshark, but I do not see any of this traffic in Kibana, or any real activity in Grafana.  I even ran tcpdump on the interface from within security onion and only see ARP requests..  

Any ideas as to how to troubleshoot this further or what I may have configured wrong?  Is there a config file I can grab and put in here to help?

**Comment by dougburks** [comment]
Have you tried running tcpdump on your other Security Onion interface just to see if perhaps the SPAN traffic might be there?

**Reply by jrsmsn** [reply]
Thank you for the reply.  It just so happens over the weekend I solved the problem.  As I mentioned, troubleshooting the issue with Wireshark allowed me to see that the packers were being seen at the end of the cable plugged in to the R610 interface, so I thought it HAD to be in the interface itself, either physically or within ESXi (didn't mentioned that was my Type 1 HV).  After digging in to the settings... i realized that vNIC was set to REJECT everything...promiscuous mode..everything.  duh!  Once I set to accept, i saw tcpdump flood with activity as soon as I hit save.  Problem solved!

**Comment by jrsmsn** [comment, marked as answer]
Checked vNIC settings in ESXi and set to accept all, problem solved.  Probably first thing I should've checked...lol.
